whereby/browser-sdk
# Add externalId as attribute in RoomConnectionOptions

https://linear.app/whereby/issue/KOA-454/add-externalid-as-an-attribute-for-the-browser-sdk

Related to recent merge in stable release: https://github.com/whereby/browser-sdk/pull/89

Adds externalId to embed element and via RoomConnectionOptions.

## Test plan
* Run SDK in a demo-app
* Add `externalId` to options in `useRoomConnection` hook
eg.

``` const roomConnection = useRoomConnection(ROOM_URL, {
    displayName: "james",
    localMediaConstraints: {
      video: true,
      audio: false,
    },
    logger: console,
    externalId: "jimbob-external-id",
  });
```
* Verify externalId is passed through to signal messages.


## Screenshots

<img width="1242" alt="Screenshot 2023-10-19 at 16 37 46" src="https://github.com/whereby/browser-sdk/assets/117101145/777306da-a338-4055-b9f0-0e4536009bd6">
